ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -62,8 +58,12 @@ class PhaserUnit extends PhaserAnimatedEntity {
 	): void {
 		super.size(data);
 		const sprite = this.sprite;
-		this.label.y = (-25 - (sprite.displayHeight + sprite.displayWidth) / 4);
-		if (this.attributesContainer) this.attributesContainer.y = 25 + (sprite.displayHeight + sprite.displayWidth) / 4;
+		if (this.label) {
```

Comment:
why not use new methods here?

Reply:
why we need create label or container here? its resizing unit and if he have label/container, change their positions accordingly.

Reply:
makes sense :)